modules/administration-guide/partials/proc_enabling-self-registration.adoc

```diff
@@ -0,0 +1,14 @@
+// configuring-authorization
+
+[id="enabling-self-registration_{context}"]
+= Enabling self-registration
+
+For {prod-short} installed without OpenShift OAuth support, self-registration of users is disabled by default, therefore the option to register a new user is not available on the login page. 
+
+.Procedure
+
+To enable self-registration of users:
+
+. Navigate to the *Realm Settings* menu on the left and open the *Login* tab. 
```

Comment:
Should we emphasize here that it has to be admin? Should do that in Keycloak? Maybe add a link on how to get a keycloak password (if that documentation is available.)?

Reply:
@skabashnyuk I've added a prerequisite that mentions that you must be logged in as an admin - https://github.com/eclipse/che-docs/pull/1819/commits/305535f261b7ee9761ee1dd0c2886ad3e3d40973